docs/pages/Configuration.md

```diff
@@ -750,6 +751,26 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 4.0
+
+**Breaking changes:**
+
+- @ui5/builder: Remove bundle option [`usePredefineCalls`](#properties) 
+- @ui5/builder: ui5-builder: New option “async“ for [builder.bundles.bundleDefinition.section](#properties)
+- @ui5/builder: Task API "namespace" option got renamed to ["projectNamespace"](https://sap.github.io/ui5-tooling/v4/api/module-@ui5_builder_tasks_bundlers_generateStandaloneAppBundle.html)
```

Comment:
This is supposed to only list changes in the Specification Version (i.e. the schema or functionality that explicitly checks the spec version).

API changes are usually independent from the Specification Version. For example this one affects projects of all Specification Versions and should therefore be mentioned in the UI5 Tooling 4.0 changelog but not here in the Specificatgion Version 4.0. Same for the rest of the chapter. See spec version 3.2 for an example.

Reply:
Thanks!